hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,15 @@
+RELEASE_TYPE: minor
+
+This release improves our support for datetimes and times around DST transitions.
+
+:func:`~hypothesis.strategies.times` and :func:`~hypothesis.strategies.datetimes`
+are now sometimes generated with ``fold=1``, indicating that they represent the
+second occurrence of a given wall-time when clocks are set backwards.
+This may be set even when there is no transition, in which case the ``fold``
+value should be ignored.
+
+For consistency, timezones provided by the :pypi:`pytz` package can now
+generate imaginary times.  This has always been the case for other timezones.
```

Comment:
could do with less than 10 words defining imaginary times, e.g.

 > generate imaginary times (i.e. the hours skipped over when DST 'springs forward')

Reply:
Though going by a docstring elsewhere in the PR, the "i.e." should become "e.g."

Reply:
Done - the release notes should now stand alone :smile: